migrations/versions/ce3e8bc1ce0f_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
```

Comment:
Migration folder should never be pushed to github. Delete it from github and and add it to gitignore.
Procedure to regenerate the migration files should be put in your readme file..